### ISSUE TITLE
Horizontal renderer

### ISSUE THREAD

**Issue by pmdserrano** [artbelikov/angular2-tree-diagram#32]
Instead of rendering it vertically, can it have an option to render horizontally?

**Comment by thxmike** [comment]
Yeah, something like that would be of high value. Even if you can supply the locations in the code to make so this can be supported, that would be awesome.
